### PR TITLE
logging: adds Honeycomb logging endpoint support

### PR DESCRIPTION
**Pull request by mccurdyc**
## Relevant (Referenced) Link(s)

* [Fastly API Docs](https://docs.fastly.com/api/logging#api-section-logging_honeycomb)
* [fastly/go-fastly](https://github.com/fastly/go-fastly/blob/c2c2c62210800daab641161412fbe210004aac0a/fastly/honeycomb.go)

## Validation

```
$ make test
```

```
git clone git@github.com:mccurdyc/cli.git /tmp/cli && \
(
    cd /tmp/cli && \
    git checkout mccurdyc/logging-honeycomb && \
    make test && \
    rm -rf /tmp/cli \
)
```

### CODE REVIEW

File: pkg/logging/honeycomb/create.go [hunk 1]
```diff
@@ -0,0 +1,74 @@
...
+
+// NewCreateCommand returns a usable command registered under the parent.
+func NewCreateCommand(parent common.Registerer, globals *config.Data) *CreateCommand {
+	var c CreateCommand
+
+	// Without this, parsing flags leads to panics.
+	// This is because if we try to scan directly into c.Input.<Field> without first
+	// creating a valid reference, flags will attempt to be scanned into null pointers.
+	// This will result in sending the zero value values for unset flags which is acceptable
+	// for Creates, but not Updates.
```
Comment (mccurdyc) [inline pkg/logging/honeycomb/create.go:29]: Good? Necessary?

**Comment by pteichman** [comment]
Conflict fix is at https://github.com/fastly/cli/commit/367815457d8693ee1cd87370dc75c8781feac6fe

**Comment by mccurdyc** [comment]
Force-pushed Peter's commit :)

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Comment by mccurdyc** [comment]
Force-pushed squashing into a single commit.

**Comment by mccurdyc** [comment]
Force-pushed squash commit of stronger tests.

**Comment by mccurdyc** [comment]
Force-pushed rebasing master.

**Pull request merged**